Makefile

```diff
@@ -527,10 +527,11 @@ docker-compose-build-swagger: awx/projects docker-compose-sources
 SCHEMA_DIFF_BASE_BRANCH ?= devel
 detect-schema-change: genschema
 	curl https://s3.amazonaws.com/awx-public-ci-files/$(SCHEMA_DIFF_BASE_BRANCH)/schema.json -o reference-schema.json
-	# Ignore differences in whitespace with -b
+  # Ignore differences in whitespace with -b
 	diff -u -b reference-schema.json schema.json
 
 docker-compose-clean: awx/projects
+  # FIXME: Check for file first
```

Comment:
I'm not really sure what this FIXME is suggesting. Could you clarify?

Reply:
Oh yeah. If the `docker-compose.yml` file isn't there, you get kind of a nasty error. But, this FIXME could apply to a ton of these rules/commands, so it might be better to just leave it out for now.